pkgs/development/haskell-modules/configuration-hackage2nix/main.yaml

```diff
@@ -31,6 +31,8 @@ default-package-overrides:
   - hiedb < 0.7
   # 2025-08-03: need to match stackage version of hosc
   - hsc3 >= 0.21 && < 0.22
+  # 2025-08-14: has to match the GHC version (currently 9.8.2)
+  - liquid-fixpoint == 0.9.6.3.3
```

Comment:
Sorry, if this has been discussed by the other discussion on this PR. But the default ghc on this branch is 9.10. Is the comment wrong? Or is that version forward compatible by accident?

I think this comment is awesome in principle, by the way, since it will be trivial us for us to see when to bump/drop the override.

Reply:
Done.